climakitae/selectors.py

```diff
@@ -350,8 +355,21 @@ def __init__(self, **params):
         self.param["variable"].objects = self.choices["variable_choices"]["hourly"][
             "Dynamical"
         ]
-        self.variable = self.default_variable
-
+        self.units = self.descrip_dict[self.variable]["native_unit"]
+        self.native_units = self.descrip_dict[self.variable]["native_unit"]
+
+    @param.depends("variable","native_units","units","descrip_dict", watch=True)
+    def _update_unit_options(self): 
+        """ Update unit options and native units for selected variable. """
+        _default_unit = self.descrip_dict[self.variable]["native_unit"]
```

Comment:
I don't like that the native_unit is read from the .csv instead of the catalog metadata. It's also not something that the user will ever actually select, so it doesn't make sense to have it as a param in DataSelector. Can you explain this choice or change it?

Reply:
I would agree with this. Had a hard time figuring out the best way to incorporate this feature into the code. I'll take a look at restructuring the code to read from the catalog instead. Thanks for your input!

Reply:
I'll let you know once I've implemented this change so you can give it another look

Reply:
These changes have now been implemented! :D